_posts/2019-11-31-mechanical-keyboard.md

```diff
@@ -11,16 +11,20 @@ image2:
 author: Valentino Urbano
 ---
 
+# Why a Split Keyboard?
+
+I've been reading a lot about [carpal tunnel syndrome][1] in the last few days and I'm trying to be mentally aware of the position of my hands while typing. On a normal keyboard you cannot keep your wrists aligned since the keys are too close to one another and either the right or the left wrist will have to tilt to reach certain letters. The solution would be to try a split keyboard or a keyboard that has some space between the right and left side of the keyboard. I know one person in the office is already using it and from what he tells me he can now type faster than on a normal keyboard.
+
+[1]: https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=2&cad=rja&uact=8&ved=2ahUKEwibvNb7143mAhWPuIsKHVKuCN4QFjABegQIAxAB&url=https%3A%2F%2Fmedium.com%2F%40samrosecruz%2Fa-developers-nightmare-carpal-tunnel-syndrome-c00f6cfaa338&usg=AOvVaw2Nq9Lq-CORpkZu6-uGTYkD
+
 # Mechanical keyboard
 
-I have just bought a mechanical custom made keyboard. It is really hard to use since it is a split keyboard without any label on any of the keys. Because of that my typing speed on it is abysmal. I'm so used to having a 'normal' keyboard that it completely threw me off. I'm also pretty fast at typing on a 'normal' keyboard without looking and I thought that it would translate into a decent performance here as well. Oh boy how wrong I was.
+This brings up to today: I have just bought a mechanical custom made keyboard. It is really hard to use since it is a split keyboard without any label on any of the keys. Because of that my typing speed on it is abysmal. I'm so used to having a 'normal' keyboard that it completely threw me off. I'm also pretty fast at typing on a 'normal' keyboard without looking and I thought that it would translate into a decent performance here as well. Oh boy how wrong I was.
```

Comment:
It really today anymore is it :P

Reply:
Well the publish date is 31/11 so the today is 'legit' imo xD